configure.in

```diff
@@ -5761,6 +5762,37 @@ if test "$wxUSE_SECRETSTORE" = "yes"; then
     fi
 fi
 
+
+dnl ---------------------------------------------------------------------------
+dnl Spellchecking for wxTextCtrl
+dnl ---------------------------------------------------------------------------
+
+if test "$wxUSE_SPELLCHECK" = "yes"; then
+    dnl The required APIs are always available under MSW and OS X but we must
+    dnl have GNOME gtkspell under Unix to be able to compile this feature.
+    if test "$wxUSE_MSW" != "1" -a "$wxUSE_OSX_COCOA" != 1; then
+        if test "$wxGTK_VERSION" != "3"; then
```

Comment:
I'm not sure if this does the right thing for the other ports, i.e. non-{GTK,MSW,OSX} ones. I think it could be better to write it as
```autoconf
if test "$wxUSE_MSW" = 1 -o test "$wxUSE_OSX_COCOA" = 1; then
    dnl Always available under these systems
    has_spellcheck_support=yes
elif test "$wxUSE_GTK" = 1; then
    PKG_CHECK_MODULES(....)
else
    dnl Not implemented in the other ports yet.
    has_spellcheck_support=no
fi

if test "$has_spellcheck_support" = yes; then
    AC_DEFINE(wxUSE_SPELLCHECK)
fi
```